test/versus_score_test.go

```diff
@@ -48,9 +49,19 @@ func TestDisjunctionSearchScoreIndexWithCompositeFields(t *testing.T) {
 
 func disjunctionQueryiOnIndexWithCompositeFields(indexName string,
 	t *testing.T) []*search.DocumentMatch {
+	tmpIndexPath, err := ioutil.TempDir("", "bleve-testidx")
```

Comment:
Why not use the same createTmpIndexPath(..)-defer cleanupTmpIndexPath(..) code path here?

Reply:
It's in a different package, and this is the only usage in this package, so I figured more clear to do it directly.  If you think copy/pasting those funcs is better, I don't feel too strongly about it.

Reply:
Fair enough :)